common/common-rest/src/main/java/org/apache/servicecomb/common/rest/codec/param/RestClientRequestImpl.java

```diff
@@ -169,6 +173,10 @@ private void attachFile(String boundary, Iterator<Entry<String, Part>> uploadsIt
     // because pojo consumer not easy to set name to part
     String name = entry.getKey();
     Part part = entry.getValue();
+    if (null == part) {
```

Comment:
Since uploads do not have null entries, this is redundant null check.

Reply:
Redundant null check has been removed. Please review again.